src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Email;
+import mate.academy.spring.validation.Password;
+
 public class UserRequestDto {
+    @Email
     private String email;
+    @Password
     private String password;
+    private String repeatPassword;
```

Comment:
this field is currently not used, add it to validation

Reply:
I forget, done